@types/pal/audio.d.ts

```diff
@@ -0,0 +1,121 @@
+        onPlay (cb): OneShotAudio;
+        /**
+         * register the end callback
+         * @param cb
+         */
+        onEnded (cb): OneShotAudio;
+    }
+    export class AudioPlayer {
+        private constructor (nativeAudio: unknown);
+        /**
+         * destroy AudioPlayer
+         */
+        destroy ();
+        /**
+         * load AudioPlayer
+         * @param url
+         * @param opts
+         */
+        static load (url: string, opts?: import('pal/audio/type').AudioLoadOptions): Promise<AudioPlayer>;
+        /**
+         * load native audio for playing one shot
+         * @param url
+         * @param opts
+         */
+        static loadNative (url: string, opts?: import('pal/audio/type').AudioLoadOptions): Promise<any>;
+        /**
+         * max audio channel, if the amount of playing audios exceeds maxAudioChannel, some audio instances should be discarded by audio manager
+         */
+        static maxAudioChannel: number;
+
+        /**
+         * type of AudioPlayer，there are WEB_AUDIO and DOM_AUDIO for web platform
+         */
+        get type (): import('pal/audio/type').AudioType;
+        /**
+         * state of AudioPlayer，restores to INIT when the audio ended playing
+         */
+        get state (): import('pal/audio/type').AudioState;
+        /**
+         * whether to loop
+         */
+        get loop (): boolean;
+        set loop (val: boolean);
+        /**
+         * volume of AudioPlayer
+         */
+        get volume (): number;
+        set volume (val: number);
+        /**
+         * duration of AudioPlayer
+         */
+        get duration (): number;
+        /**
+         * read only current time of AudioPlayer, if you want to set the current time, please call the seek() method
+         */
+        get currentTime (): number;
+        /**
+         * seek the currentTime，the returned Promise is to ensure the completion of asynchronous operation
+         * @param time
+         */
+        seek (time: number): Promise<void>;
+        /**
+         * play one shot of the audio，the returned OneShotAudio can be managed by audio manager
+         * @param volume
+         */
+        playOneShot (volume?: number): OneShotAudio;
+        /**
+         * play audio or resume audio when it is paused, the returned Promise is to ensure the completion of asynchronous operation
+         */
+        play (): Promise<void>;
+        /**
+         * pause audio, the returned Promise is to ensure the completion of asynchronous operation
+         */
+        pause (): Promise<void>;
+        /**
+         * stop audio, the returned Promise is to ensure the completion of asynchronous operation
+         */
+        stop (): Promise<void>;
+
+        /**
+         * register the InterruptionBegin callback，the Interruption includes the show/hide events, phone call/alarm, unpluging earphones
+         * @param cb
+         */
+        onInterruptionBegin (cb: () => void);
```

Comment:
These methods do not have removal edition right?

Reply:
`offInterruptionBegin` should be the `removal edition` you mention